samples/drivers/crypto/prj_stm32.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_LOG=y
+CONFIG_MAIN_STACK_SIZE=4096
+
+CONFIG_CRYPTO=y
+CONFIG_CRYPTO_STM32=y
```

Comment:
This line should not be required as CONFIG_CRYPTO_STM32 should be enabled when CONFIG_CRYPTO is enabled

Reply:
This line has been removed.